lte/gateway/c/session_manager/SessionProxyResponderHandler.cpp

```diff
@@ -88,34 +96,54 @@ void SessionProxyResponderHandlerImpl::PolicyReAuth(
     std::function<void(Status, PolicyReAuthAnswer)> response_callback) {
   auto& request_cpy = *request;
   PrintGrpcMessage(static_cast<const google::protobuf::Message&>(request_cpy));
-  MLOG(MDEBUG) << "Received a Gx (Policy) ReAuthRequest for session_id "
-               << request->session_id();
   enforcer_->get_event_base().runInEventBaseThread([this, request_cpy,
                                                     response_callback]() {
     PolicyReAuthAnswer ans;
-    auto session_map = session_store_.read_sessions({request_cpy.imsi()});
+    const std::string& imsi = request_cpy.imsi();
+    const std::string& session_id = request_cpy.session_id();
+    SessionSearchCriteria criteria(imsi, IMSI_AND_SESSION_ID, session_id);
+    auto session_map = session_store_.read_sessions({imsi});
+    auto session_it = session_store_.find_session(session_map, criteria);
+    auto& session = **session_it;
+
+    const auto& config = session->get_config();
+    const auto& common_context = config.common_context;
+
     SessionUpdate update =
         SessionStore::get_default_session_update(session_map);
-    enforcer_->init_policy_reauth(session_map, request_cpy, ans, update);
-    MLOG(MDEBUG) << "Result of Gx (Policy) ReAuthRequest "
-                 << raa_result_to_str(ans.result());
+    const auto rat_type = common_context.rat_type();
+    const std::string& interface = (rat_type == TGPP_NR) ? "5G N7" : "4G Gx";
+    MLOG(MDEBUG) << "Received a "
+                 << interface << " (Policy) ReAuthRequest for session_id "
+                 << session_id;
```

Comment:
I think this log was better placed before. If we receive a RAR we want to know we receive it. 
What will happen if the session is not found on sessiond? If you were to exit, this message would never be printed.

Also check in other function for the placement of this message to be consistent. Should it be before `runInEventBaseThread` or inside the `runInEventBaseThread`